doc/api/deprecations.md

```diff
@@ -582,6 +582,19 @@ deprecated.
 *Note*: `OutgoingMessage.prototype._renderHeaders` was never documented as
 an officially supported API.
 
+<a id="DEP0068"></a>
+### DEP0068: Usage of _stream_*
```

Comment:
For now, make this `DEP00XX', we can update the number when it lands, that way there's no risk of conflict